### ISSUE TITLE
Rename/refactor context menu options

### ISSUE THREAD

**Issue by alexsb** [visdesignlab/data-hunch#14]
<img width="192" alt="image" src="https://user-images.githubusercontent.com/2039375/159459556-a7613fb6-150a-4424-bd82-17cede2642d1.png">

* This shouldn't be caps, but regular case. 
* A context menu doesn't have an X to close it 
* The text should be left-aligned.

Here are different labels in a different order:

Manipulation -> Change size
Range -> Specify Range or Values
Categorical -> Change Type/Category
Exclusion -> Remove item
Data Space -> Change value
Annotation -> Add comment
Rating -> Rate item

**Comment by haihan-lin** [comment]
I would think we want to use the vocabulary close to what we use in the paper.

Also manipulation -> change size ? I am not sure there is a different between change size and change value

**Comment by haihan-lin** [comment]
![WeChata3d8c3fbd3da466a18ffc7b5f2fa7052](https://user-images.githubusercontent.com/14115944/159590940-2fa971b3-2737-4d40-97e4-dc81df309f9e.png)

Progress: All done besides label change

**Comment by alexsb** [comment]
Your argument with using paper language makes sense, though I'm a bit worried that it might not be clear what what does if you use the tool without having read the paper. 

Maybe we should use simple terms I suggested and have the paper term (if it deviates) in brackets? So "Change Size (Manipulation)"?

And Change Size was a conscious decision, because it applies to visual space - you're actually changing the size of the bar, not of the value that drives the size (which is "change value"). I'd recommend sticking with it, but combining it with the paper terms so it's not ambiguous. 

Finally, the order I had suggested was purposeful. Mostly, I don't want the relatively boring "Annotation" to be the first option.
